src/audio/pipeline/pipeline-schedule.c

```diff
@@ -70,7 +70,6 @@ static enum task_state pipeline_task_cmd(struct pipeline *p,
 		}
 
 		if (err == PPL_STATUS_PATH_STOP) {
-			pipe_warn(p, "pipeline_task_cmd(): stopping for xrun");
```

Comment:
What are the other cases here ? do we need to spell them out in a comment ?

Reply:
comment added